### PR TITLE
More ImagePath tests

### PR DESCRIPTION
**Pull request by Yay295**

### CODE REVIEW

**Comment by radarhere** [comment]
This is another extract from #6653, yes?

**Comment by Yay295** [comment]
yes

File: Tests/test_imagepath.py [hunk 1]
```diff
@@ -39,48 +38,76 @@ def test_path():
...
+        [0, 1],
+        (0.0, 1.0),
+        [0.0, 1.0],
+        ((0, 1),),
+        [(0, 1)],
+        ((0.0, 1.0),),
+        [(0.0, 1.0)],
+        array.array("f", [0, 1]),
+        array.array("f", [0.0, 1.0]),
+        ImagePath.Path((0, 1)),
```
Comment (radarhere) [inline Tests/test_imagepath.py:55]:
```suggestion
        ImagePath.Path(array.array("f", [0, 1]).tobytes()),
```
A suggestion, to keep the original test case of
```python
arr = array.array("f", [0, 1])
p = ImagePath.Path(arr.tobytes())
```
intact.

Comment (Yay295) [reply Tests/test_imagepath.py:55]: Line 54 should change, not 55. The new line 55 is for the original line 51.

Comment (radarhere) [reply Tests/test_imagepath.py:55]: Right, thanks

**Comment by jrmendeshurb4** [comment]
ok

**Comment by jrmendeshurb4** [comment]
ok

**Comment by jrmendeshurb4** [comment]
ok

**Review by hugovk — APPROVED** [review]

**Pull request merged**